content/en/containers/datadog_operator/crd_dashboard.md

```diff
@@ -0,0 +1,97 @@
+---
+title: DatadogDashboard CRD
+---
+To deploy a Datadog dashboard, you can use the Datadog Operator and `DatadogDashboard` custom resource definition (CRD).
+
+### Prerequisites
+- [Helm][1]
```

Comment:
```suggestion
- [Helm][1]
```
Just curious is there any minimum Helm version required to use this CRD ?

Reply:
I don't believe so!